### ISSUE TITLE
Topical diclofenac is not an ancestor of NSAIDs

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Athena#264]
For some reason concepts for topical diclofenac are not considered NSAIDs:

```sql
SELECT * 
FROM concept_ancestor
WHERE ancestor_concept_id = 21603933 --ANTIINFLAMMATORY AND ANTIRHEUMATIC PRODUCTS, NON-STEROIDS
  AND descendant_concept_id = 40162333 --diclofenac sodium 15 MG/ML Topical Solution;
-- 0 rows
```

Possibly this is because the generic concept for diclofenac solutions isn't a descendant of NSAIDs:

```sql
SELECT * 
FROM concept_ancestor
WHERE ancestor_concept_id = 21603933 --ANTIINFLAMMATORY AND ANTIRHEUMATIC PRODUCTS, NON-STEROIDS
  AND descendant_concept_id = 40032426 --diclofenac Topical Solution;
-- 0 rows
```

Note that the specific and generic diclofenac topical solution concepts are descendants of diclofenac:

```sql
SELECT * 
FROM concept_ancestor
WHERE ancestor_concept_id = 1124300 --diclofenac
  AND descendant_concept_id = 40162333 --diclofenac sodium 15 MG/ML Topical Solution;
-- 1 row

SELECT * 
FROM concept_ancestor
WHERE ancestor_concept_id = 1124300 --diclofenac
  AND descendant_concept_id = 40032426 --diclofenac Topical Solution;
-- 1 row
```

**Comment by cgreich** [comment]
@schuemie:

That's not a bug, it's a feature of ATC. As the name suggests, there are three hierarchical category mechanisms: Anatomic, therapeutic (indication) and chemical (ingredient), in that order. RxNorm, and the major drug hierarchies, are organized as chemical, form, strength, brand.

In your case, topical diclofenac preparations belong to ATC5 concept_id 21602355 "diclofenac; topical (dermatologicals)". Which means their main hierarchical category is "Skin" (=dermatologicals), rather than "Musculoskeletal" for the systematic NSAIDS (there is actually a third one, "Sensory Organs" for the ophthalmic diclofenac).

Bottom line: Every ingredient (chemical) can be and often is more than once in ATC, and you have to follow them up in the hierarchy to see whether and where they converge, and whether you want to include all of them.

BTW: to make that easier to spot we already are tweaking the concept_name and adding the inherited A and T information. In the original WHO ATC, your topical diclofenac would just be called "[diclofenac](https://www.whocc.no/atc_ddd_index/?code=M02AA15)". Nice, eh?

Two questions:
- Can we close this?
- Should we have this conversation on the Forum? Sounds like others would benefit.

**Comment by schuemie** [comment]
Oops! I should have know that. Thanks!

I'll close this. 

I don't think the Forum is the right place for documentation. Do we have vocab user documentation somewhere?

**Comment by mik-ohdsi** [comment]
Hi @schuemie , we do have vocab user documentation, but it is still evolving. Here is the [general list](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Standardized-Vocabularies) and here is [ATC](https://github.com/OHDSI/Vocabulary-v5.0/wiki/Vocab.-ATC).

**Comment by schuemie** [comment]
Hi @mik-ohdsi . This documentation is excellent! I think it might be helpful to also have some generic help on the vocabulary (e.g. how the concept_ancestor table should be used). With regards to the issue I raised above, it might be helpful to explicitly mention that a single ingredient can correspond to multiple ATC codes, and that drugs containing the ingredient will correspond to some but not all of those ATC codes.
